domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/TimelineEvent.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.hibernate.validator.constraints.NotBlank;
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+public class TimelineEvent {
+
+    @NotBlank
+    @Size(max = 25)
+    private final String date;
+
+    @NotBlank
```

Comment:
would be nice if these were sorted see above,
also doesn't `NotBlank` imply `NotNull`?